frame/support/test/tests/construct_runtime.rs

```diff
@@ -254,7 +254,7 @@ impl system::Config for Runtime {
 }
 
 frame_support::construct_runtime!(
-	pub enum Runtime where
+	pub struct Runtime where
```

Comment:
`pub enum Runtime` is still supported so we should keep the old test and add new one for `pub struct Runtime`